e2e/run

```diff
@@ -156,13 +163,17 @@ customize_cluster_yaml() {
 worker:
   nodePools:
   - name: asg1
+    clusterAutoscalerSupport:
+      enabled: true
```

Comment:
We should add a validation which emits an error when there are no worker node pool or controller node whose `clusterAutoscalerSupport` is enabled.
Otherwise cluster-autoscaler is unable to work/be scheduled due to missing IAM permissions.